contracts/RewardController.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MIT
+// Disclaimer https://github.com/hats-finance/hats-contracts/blob/main/DISCLAIMER.md
+
+pragma solidity 0.8.6;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "./HATVaults.sol";
+
+contract RewardController is OwnableUpgradeable {
+
+    struct PoolUpdate {
+        uint256 blockNumber;// update blocknumber
+        uint256 totalAllocPoint; //totalAllocPoint
+    }
+
+    uint256 public constant MULTIPLIERS_LENGTH = 24;
```

Comment:
rename to NUM_OF_EPOCHS_IN_REWARD_PLAN or something like that? :)

Reply:
honestly not even sure why we need this thing in the first place. Let's just leave it here and address separately: https://github.com/hats-finance/hats-contracts/issues/194